README.md

```diff
@@ -17,6 +17,8 @@ The easiest way to run Sugarizer is to use Docker. To do that, type command line
 	sh generate-docker-compose.sh
 	docker-compose up -d
 
+By default, your sugarizer server will be accessible on http://127.0.0.1:8080 and your public IP. 
```

Comment:
"and your public IP" does not makes sense. It would be more appropriate to specify that by default it will be available on port 8080.

Also, make use of markdown for links such as:
"By default, Sugarizer Server UI will be accessible on [http://127.0.0.1:8080](http://127.0.0.1:8080)"